addons/binding/org.openhab.binding.solaredge/src/main/java/org/openhab/binding/solaredge/internal/command/SolarEdgeCommand.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solaredge.internal.command;
+
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.api.Response.CompleteListener;
+import org.eclipse.jetty.client.api.Response.ContentListener;
+import org.eclipse.jetty.client.api.Response.FailureListener;
+import org.eclipse.jetty.client.api.Response.SuccessListener;
+import org.openhab.binding.solaredge.internal.connector.StatusUpdateListener;
+
+/**
+ * public interface for all commands
+ *
+ * @author Alexander Friese - initial contribution
+ *
+ */
+public interface SolarEdgeCommand extends SuccessListener, FailureListener, ContentListener, CompleteListener {
+
+    public static int MAX_RETRIES = 5;
```

Comment:
interface consts are always public static so you can leave that out.

Reply:
fixed it